content/post/52-go-ipfs-0.4.18.md

```diff
@@ -0,0 +1,283 @@
+  multiplexed connection. In theory (not yet in practice), this should
+  significantly reduce the latency of DHT queries.
+* **Behaves better on lossy networks**. When multiplexing multiple requests over a
+  single TCP connection, a single dropped packet will bring the entire
+  connection to a halt while the packet is re-transmitted. However, because QUIC
+  handles multiplexing internally, dropping a single packets affects only the
+  related stream.
+* **Better NAT traversal**: NAT hole-punching is significantly easier and, in
+  many cases, more reliable with UDP than with TCP.
+
+However, we still have a long way to go. While we encourage users to test this,
+the IETF QUIC protocol is still being actively developed and *will* change. You can find instructions for enabling it [here](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#QUIC).
+
+## Pubsub
+
+go-ipfs now supports the [gossipsub](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) routing algorithm and message signing.
+
+The gossipsub routing algorithm is *significantly* more efficient than the
+current floodsub routing algorithm. Even better, it's fully backwards compatible
+so you can enable it and still talk to nodes using the floodsub algorithm. You
+can find instructions to enable gossipsub in go-ipfs
+[here](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#gossipsub).
+
+Messages are now, finally, **signed by their authors**. While signing has been
+enabled by default, strict signature verification has not been and will not be
+for at least one release (probably multiple) to avoid breaking existing
+applications. You can read about how to configure this feature
+[here](https://github.com/ipfs/go-ipfs/blob/master/docs/experimental-features.md#message-signing).
+
+## Commands Changes
+
+In terms of new toys, this release introduces the `ipfs cid` subcommand for
+working with CIDs, a completely refactored `ipfs p2p` command, streaming name
+resolution, and complete inline block support.
+
+#### `ipfs cid`
+The new `ipfs cid` command allows users to both inspect CIDs and convert them
+between various formats and versions. Here are some examples:
+
+```sh
+# Print out the CID metadata (prefix)
+> ipfs cid format -f %P QmT78zSuBmuS4z925WZfrqQ1qHaJ56DQaTfyMUF7F8ff5o
+cidv0-protobuf-sha2-256-32
+
+# Get the hex sha256 hash from the CID.
+> ipfs cid format -b base16 -f '0x%D' QmT78zSuBmuS4z925WZfrqQ1qHaJ56DQaTfyMUF7F8ff5o
+0x46d44814b9c5af141c3aaab7c05dc5e844ead5f91f12858b021eba45768b4c0e
+
+# Convert a base58 v0 CID to a base32 v1 CID.
+> ipfs cid base32 QmT78zSuBmuS4z925WZfrqQ1qHaJ56DQaTfyMUF7F8ff5o
+bafybeicg2rebjoofv4kbyovkw7af3rpiitvnl6i7ckcywaq6xjcxnc2mby
+```
+
+#### `ipfs p2p`
+The refactored `ipfs p2p` command allows forwarding TCP streams through two IPFS
+nodes from one host to another. It's `ssh -L` but for IPFS. 
+It's still experimental but we don't expect too many breaking changes after this
```

Comment:
Lets underline "it's still experimental" - right now it's getting lost in the paragraph and it's important context!

Reply:
I don't think github supports underline in markdown. I'll bold it.